### PR TITLE
Removes monthly report for versions before 1.3

### PR DESCRIPTION
**Pull request by NejcZdovc**
Resolves https://github.com/brave/brave-browser/issues/8987

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
- start browser with a profile that has history of rewards before 2020
- open monthly statement
- make sure that you only see Feb 2020 and forward

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: components/brave_rewards/resources/page/components/pageWallet.tsx [hunk 1]
```diff
@@ -676,7 +676,15 @@ class PageWallet extends React.Component<Props, State> {
         return
       }
 
-      const date = new Date(Date.UTC(parseInt(items[0], 10), parseInt(items[1], 10) - 1, 10))
+      const year = parseInt(items[0], 10)
+      const month = parseInt(items[1], 10) - 1
+
+      // we only want to show reports from version 1.3 (Feb 2020) up
```
Comment (tmancey) [inline components/brave_rewards/resources/page/components/pageWallet.tsx:682]: 1.3 was available from 13th November 2019?

**Comment by NejcZdovc** [comment]
@tmancey 1.3 was released February 7, 2020. Nightly was Nov 2019

**Comment by tmancey** [comment]
> @tmancey 1.3 was released February 7, 2020. Nightly was Nov 2019

1.3.1 was released to "nightly" in November, however, @NejcZdovc has clarified it was decided that this fix should only consider "release"

**Comment by NejcZdovc** [comment]
CI failed on a known thing

**Comment by LaurenWags** [comment]
Verified passed with

Brave | 1.9.21 Chromium: 81.0.4044.92 (Official Build) nightly (64-bit)
-- | --
Revision | 32921c79b6f01a0fb2deef0e1d45b42f96581051-refs/branch-heads/4044@{#883}
OS | macOS Version 10.14.6 (Build 18G3020)

Personal profile monthly statement on 1.8.75 (Beta):
Screen Shot 2020-04-14 at 1 57 32 PM

Personal profile using 1.9.21 (Nightly)
Screen Shot 2020-04-14 at 2 06 26 PM

**Pull request merged**